content/_layouts/post.html.haml

```diff
@@ -42,6 +41,3 @@ layout: default
               = display_author_for(page)
 
         = content
-
-      %div#sidebar.col-md-4.events.vertical
-        = partial('sidebar.html.haml')
```

Comment:
I'll be creating a real fix tomorrow. This is a stop gap so that a few
hundred people on mobile can read the site.
On Fri, May 27, 2016 at 7:04 PM, Manuel Recena <notifications@github.com>
wrote:

> In content/_layouts/post.html.haml
> <https://github.com/jenkins-infra/jenkins.io/pull/269#discussion_r64877069>
> :
>
> > @@ -42,6 +41,3 @@ layout: default
> >                = display_author_for(page)
> >
> >          = content
> > -
> > -      %div#sidebar.col-md-4.events.vertical
> > -        = partial('sidebar.html.haml')
>
> 🐛 and 👎
>
> This is a complete hack for a specific post.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/269/files/52c668315a4f3918157d662b835841cdfca46efe#r64877069>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADD7MQspX7g-4EmZWTVJrwN5-2OGO_Uks5qFrOqgaJpZM4IoOs5>
> .
>
-- 
James Dumay
Product Manager
CloudBees, Inc.
Skype: ap_myskype Twitter: i386